akka-projection-grpc-tests/src/it/scala/akka/projection/grpc/producer/EventProducerPushSpec.scala

```diff
@@ -145,12 +146,15 @@ class EventProducerPushSpec(testContainerConf: TestContainerConf)
           .withInterceptor((_, metadata) =>
             if (metadata.getText("secret").contains("password")) Future.successful(Done)
             else throw new GrpcServiceException(Status.PERMISSION_DENIED))
+          // FIXME why not just StringValue.scalaDescriptor for scala API?
+          .withProtobufDescriptors(StringValue.javaDescriptor.getFile :: Nil)
```

Comment:
not sure, but could be something we inherited from Kalix. pls create an issue for follow up

Reply:
https://github.com/akka/akka-projection/issues/958